examples/redwoodjs/redwood.toml

```diff
@@ -0,0 +1,7 @@
+[web]
+  port = 8910
```

Comment:
Is it possible to change the port here to use the `$PORT` environment variable? We need a way for `vercel dev` to assign the port dynamically.

Reply:
I’m not sure—there’s a decent chance we’ll need to just talk to the Redwood team directly, they haven’t optimized for anything but Netlify yet